common/src/main/java/com/microsoft/hydralab/common/file/impl/azure/AzureBlobProperty.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+package com.microsoft.hydralab.common.file.impl.azure;
+
+import com.microsoft.hydralab.common.file.ContainerConstants;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author zhoule
+ * @date 11/15/2022
+ */
+@Data
+@ConfigurationProperties(prefix = "app.storage.blob")
+@Component
+public class AzureBlobProperty implements ContainerConstants {
+    private String connection;
+    private long SASExpiryTimeFront;
+    private long SASExpiryTimeAgent;
+    private long SASExpiryUpdate;
+    private String timeUnit;
+    private int fileLimitDay;
+    private String CDNUrl;
+
+    private String SCREENSHOT_CONTAINER_NAME = "images";
```

Comment:
```suggestion
    private final static String SCREENSHOT_CONTAINER_NAME = "images";
```
same applies to all below.

Reply:
Resolved.